.vscode/settings.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Makes sense but these settings should remain rather local to not interfere with user's local settings.